app/assets/javascripts/app/print-recovery-code.js

```diff
@@ -1,5 +1,3 @@
-import isMobileUserAgent from './utils/mobile-user-agent';
```

Comment:
is this js file used elsewhere? based on a cursory search, seems like maybe not. If not, we can remove it! 🐭